### DISCUSSION TITLE
Can not open some PDF files (even in Fiddle)

### DISCUSSION THREAD

**Discussion by cristianomoraes** [hrynko/vue-pdf-embed#74] [Q&A]
Hi,

I am looking for some Vue components to find one that actually works to show PDF documents, and so far this is the only one that seems to work, at least in some situations.

It looks like it works fine with the document used in the jsFiddle examples but two other documents that I tried didn´t work:
http://www.africau.edu/images/default/sample.pdf
https://grupogpc.com.br/wp-content/uploads/2022/07/MERCADO-DE-PULGAS-DO-25o-OPEN-GPC-MODELO-COMPLETO.pdf (this is actually what I want to embed)

It shows a message: "Warning: DocException - expected a valid Error." and doesn't show the document. That is happening even in the examples in the Fiddle.

Is there something I can do to make this work?

Thank you.

**Comment by hrynko** [comment]
Hi @cristianomoraes,

Just checked these sample PDFs and experienced the same issue. I also tried to reproduce this in other libraries and in PDFJS itself - it's the same everywhere.

```json
{
  "message": "Failed to fetch",
  "name": "UnknownErrorException",
  "details": "TypeError: Failed to fetch"
}
```

Not sure how to handle this yet as the error does not provide any insights.

**Reply by hrynko** [reply]
I did a little more research and now I think the problem is the server, not the file. To handle this, you can try loading the file manually (via JS) and then passing to `vue-pdf-embed` as a `base64` string. That worked for me.

**Reply by cristianomoraes** [reply]
Hi @hrynko. I did try this solution in my application and didn't work yet, but I think there are some issues in my code. I will have a look.
Thank you.

**Comment by cristianomoraes** [comment]
Thank you for having a look at this @hrynko.
I will try to find some time to investigate this too.
Cheers
